docs/sources/tempo/configuration/sidecar-proxy.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Configure sidecar proxies
+menuTitle: Configure sidecar proxies
+description: Configure Tempo Distributed to run with sidecar proxies
+weight: 36
+---
+
```

Comment:
Since this is optional, I think it makes sense to have an intro about that when to use it.  Kind of similar to the `Some installations require...` part below, so maybe updated that part as well.

Reply:
@mdisibio When would you suggest this procedure should be used?